mega.go

```diff
@@ -531,6 +531,9 @@ func (m *Mega) login(email string, passwd string) error {
 
 	msg[0].Cmd = "us"
 	msg[0].User = email
+	if multiFactor != "" {
```

Comment:
The if statement isn't necessary here since the default for `msg[0].Mfa` will be `""`